src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class ColorSupplier {
+    private final Color[] colors = Color.values();
+    private final Random random = new Random();
+
+    public Color getRandomColor() {
+        Color[] colors = Color.values();
+        Color randomColor = colors[random.nextInt(colors.length)];
+        return Color.valueOf(randomColor.toString());
```

Comment:
why do you unwrap and inline wrap it again?
```suggestion
        return colors[random.nextInt(colors.length)];
```